test/unit/spec/v1/LoginRouter_spec.js

```diff
@@ -1309,30 +1319,6 @@ Expect.describe('v1/LoginRouter', function() {
       });
     });
 
-    itp('can redirect with PKCE flow', function() {
-      return setupOAuth2({
-        'redirect': 'always'
-      }, { mockWellKnown: true, expectRedirect: true }).then(
-        expectCodeRedirect({
-          responseType: 'code',
-          code_challenge_method: 'S256',
-        })
-      );
-    });
-
-    itp('can redirect with PKCE flow and responseMode "fragment"', function() {
-      return setupOAuth2({
-        'redirect': 'always',
-        'authParams.responseMode': 'fragment',
-      }, { mockWellKnown: true, expectRedirect: true }).then(
-        expectCodeRedirect({
-          responseType: 'code',
-          responseMode: 'fragment',
-          code_challenge_method: 'S256',
-        })
-      );
-    });
```

Comment:
same question regarding deleted tests from this file

Reply:
These tests were moved into testcafe as well, see: https://github.com/okta/okta-signin-widget/pull/3418/files#diff-1987f2296672040ace394ee49f6a4aef3ed04f56f3e8759f3fda524609c338ceR91-R150